### DISCUSSION TITLE
大佬们，可否添加一个增加头部注释的插件功能

### DISCUSSION THREAD

**Discussion by wxzvim** [ayamir/nvimdots#879] [Ideas]
写程序时，，作者、时间这些头部注释可以增加个嘛？这样感觉比较方便

**Comment by Jint-lzxy** [comment]
u may create custom snippets for this. We have no plan to implement such feature as styles used for comments vary from language to language. [Examples](https://github.com/ayamir/nvimdots/tree/main/snips/snippets)
